src/Components/ErrorMessage/ErrorMessage.tsx

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
You've disabled a11y controls in your code. Accessibility is important for your application and you should not disable it unless necessary. It's better to add an aria-label attribute to your button.